src/Helpers/authorizationFunctions.ts

```diff
@@ -1,14 +1,15 @@
 import useAuth from 'Hooks/useAuth';
 
+export interface AnyFunctions {
+  [key: string]: (...args: any[]) => boolean; // NOSONAR
+}
+
 const IsAuthenticated = () => {
+  // NOSONAR typescript:S100
```

Comment:
??

Reply:
According to React, name of function should be in PascalCase.
According to Sonar, name should be camelCase. I ignored sonar one.
I mentioned sonar rule as further